charts/harvester-csi-driver/values.yaml

```diff
@@ -6,7 +6,7 @@ image:
   harvester:
     csiDriver:
       repository: rancher/harvester-csi-driver
-      tag: ""
+      tag: "v0.1.1"
```

Comment:
If the tag is specified here, I think we should omit the default version in https://github.com/harvester/charts/blob/153951f5eccc352a812afa74dd4f74b3b0420997/charts/harvester-csi-driver/templates/daemonset.yaml#L57

Reply:
nit: 
```suggestion
      tag: v0.1.1
```